### PR TITLE
Replace fabric8 docker container by testcontainers

### PR DESCRIPTION
**Pull request by HJK181**
As requested in [PR 118](https://github.com/dadoonet/elasticsearch-beyonder/pull/118).

### CODE REVIEW

**Comment by dadoonet** [comment]
Could you tell why would you like to replace fabric8 (and launching elasticsearch externally) by testcontainers?

At the very least, if you really want to have testcontainers, I'd add it in addition to fabric8.

Like what I did in this demo project: https://github.com/dadoonet/elasticsearch-integration-tests/blob/04-check-if-already-running/src/test/java/fr/pilato/talk/elasticsearch/tests/integration/ElasticsearchIT.java#L69-L83

File: pom.xml [hunk 1]
```diff
@@ -167,6 +161,13 @@ when an elasticsearch client starts.
             <artifactId>commons-text</artifactId>
             <version>1.8</version>
         </dependency>
+        
+        <dependency>
+			<groupId>org.testcontainers</groupId>
```
Comment (dadoonet) [inline pom.xml:166]: Wrong indentation. Could you fix it?

File: pom.xml [hunk 2]
```diff
@@ -167,6 +161,13 @@ when an elasticsearch client starts.
...
+			<artifactId>elasticsearch</artifactId>
+			<version>1.14.2</version>
```
Comment (dadoonet) [inline pom.xml:168]: I'd prefer having the TC version as a property at the beginning of the `pom.xml`. Like 

```xml
<testcontainers.version>1.14.2</testcontainers.version>
```

**Comment by dadoonet** [comment]
Could you also use spaces instead of tabs for the code? And rollback the changes that were not needed (like code formatting)?

**Comment by HJK181** [comment]
I'm going to close the PR as I don't see any value in using both, testcontainers and fabric8. Will rework #118 to work with fabric8.

**Pull request closed without merging**